timesketch/api/v1/resources/nl2q.py

```diff
@@ -222,5 +222,5 @@ def post(self, sketch_id):
             )
             return jsonify(result_schema)
 
-        result_schema["query_string"] = prediction.strip("```")
+        result_schema["query_string"] = prediction.strip("`\n\r\t ")
```

Comment:
How does this solution compare to e.g. using ```prediction.replace("`", "")```?

Reply:
It would assure it does not remove the "`" if it's legitimate part of the query. Strip is used to remove characters from the beginning (leading) and end (trailing) of a string.